src/algebra/is_smul_regular.lean

```diff
@@ -0,0 +1,182 @@
+/-
```

Comment:
Since the other file is called `regular` and not `is_regular`, I'd argue this should be `smul_regular` not `is_smul_regular`

Reply:
also changed the name of the file.